src/modules/palnagotchi/pwngrid.cpp

```diff
@@ -0,0 +1,252 @@
+#ifdef CARDPUTER
+#include "pwngrid.h"
+
+uint8_t pwngrid_friends_tot = 0;
+pwngrid_peer pwngrid_peers[255];
```

Comment:
The size of this object consumes all DRAM available for StickCPlus2.

It is happening probably because:
```
	-DBOARD_HAS_PSRAM
	-mfix-esp32-psram-cache-issue
	-mfix-esp32-psram-cache-strategy=memw
```
Is used in build_flags on platformio.ini

Reducing it from 255 to 50 fix the problem, so the question is: Is is really needed to have 50 pwngrid_peers??

Reply:
I Tried without these flags, and didn´t work :(

The way I see is to limit it to 50 peers